config/manifest/eksa-components.yaml

```diff
@@ -6328,6 +6341,18 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - cluster.x-k8s.io
+  resources:
+  - machinehealthchecks
```

Comment:
setting the namespace to eksa-system converts it from a clusterrole to a role and the controller errors saying patch is forbidden.

Reply:
Do you know why that happens?
I'm not sure if it's wise to just give the controller more permissions because "there is an error" if we don't understand what error is coming from.

Reply:
Afaik this happens because the controller is cluster scoped (via the cluster roles) and can not list based on namespaces.  Looks like this is true for all other resources listed for the cluster controller as well.

Reply:
Let's take this offline, but you are trying to do "create + update" operations, not list. So that doesn't really explain the issue.

Reply:
the exact operation is patch, and I said list as an example of the operation. I believe its for an operation the client tries to perform.

Reply:
The permissions are using the ClusterRole due to client trying to list this resource from cache.
When trying to list resources not already in cache, it starts an informer for that type using the scope of the cache.
So if the manager is cluster-scoped, the new informers created by the cache will be cluster-scoped. Discussed this with Guillermo offline and came to the conclusion that the permissions need to be cluster scoped.